src/utils/dict.js

```diff
@@ -173,6 +178,8 @@ export const translateInterestPaymentDue = (interestPaymentDue) => interestPayme
 
 export const translateRepaymentType = (repaymentType) => repaymentTypeDict[repaymentType]
 
+export const translateloanPurpose = (loanPurpose) => loanPurposeDict [loanPurpose]
```

Comment:
And this unneeded space before the square bracket...

Reply:
updates